src/server/auth/server/testing/admin_test.go

```diff
@@ -582,7 +582,7 @@ func TestRobotUserACL(t *testing.T) {
 	// test that alice can commit to the robot user's repo
 	commit, err := aliceClient.StartProjectCommit(pfs.DefaultProjectName, repo, "master")
 	require.NoError(t, err)
-	require.NoError(t, aliceClient.FinishProjectCommit(pfs.DefaultProjectName, repo, commit.Branch.Name, commit.ID))
+	require.NoError(t, aliceClient.FinishProjectCommit(pfs.DefaultProjectName, repo, "", commit.ID))
```

Comment:
Should `client.FinishProjectCommit` no longer take a branch name?

No need to deprecate it in favour of a new method because it hasn’t been released yet.

Reply:
it should still take a branch name, because as a user I should still be able to "reference" a commit based on a branch

Reply:
I _think_ that we should have just taken an "identifier" field that we figure out whether to interpret as a UUID or branch name. I can't think of a situation where the client is meaningfully expressing something by passing both a branch name and a commit